pkg/util/testing/wrappers.go

```diff
@@ -93,6 +93,48 @@ func (j *JobWrapper) AddResource(r corev1.ResourceName, v string) *JobWrapper {
 	return j
 }
 
+type QueuedWorkloadWrapper struct{ kueue.QueuedWorkload }
+
+// MakeQueuedWorkload creates a wrapper for a QueuedWorkload with a single
+// pod with a single container.
+func MakeQueuedWorkload(name, ns string) *QueuedWorkloadWrapper {
+	return &QueuedWorkloadWrapper{kueue.QueuedWorkload{
+		ObjectMeta: metav1.ObjectMeta{Name: name, Namespace: ns},
+		Spec: kueue.QueuedWorkloadSpec{
+			Pods: []kueue.PodSet{
+				{
+					Count: 1,
+					Spec: corev1.PodSpec{
+						Containers: []corev1.Container{
+							{
+								Name: "c",
+								Resources: corev1.ResourceRequirements{
+									Requests: make(corev1.ResourceList),
+								},
+							},
+						},
+					},
+					AssignedFlavors: make(map[corev1.ResourceName]string),
+				},
+			},
+		},
+	}}
+}
+
+func (w *QueuedWorkloadWrapper) Obj() *kueue.QueuedWorkload {
+	return &w.QueuedWorkload
+}
+
+func (w *QueuedWorkloadWrapper) Request(r corev1.ResourceName, q string) *QueuedWorkloadWrapper {
```

Comment:
can we unify this with the AddResource func used in the job wrapper, use either name, but lets just make them the same.

Reply:
Done